src/gigs-board/components/molecule/markdown-viewer.jsx

```diff
@@ -0,0 +1,54 @@
+const Wrapper = styled.div`
+  p {
+    white-space: pre-line; // This ensures text breaks to new line
+
+    span {
+      white-space: normal; // and this ensures profile links look normal
+    }
+  }
+
+  blockquote {
+    margin: 1em 0;
+    padding-left: 1.5em;
+    border-left: 4px solid #ccc;
+    color: #666;
+    font-style: italic;
+  }
+  pre {
+    background-color: #f4f4f4;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    padding: 1em;
+    overflow-x: auto;
+    font-family: "Courier New", Courier, monospace;
+  }
```

Comment:
I would consider to contribute these styles to the [Social VM Markdown component](https://github.com/NearSocial/VM/blob/master/src/lib/components/Markdown.js) if you will have time for that.

Reply:
Good suggestion, but I don't think the VM should impose styles. The Markdown component is only meant to resolve the correct element tags. Developers can fork their own wrapper component w/ styles.

Reply:
Well, maybe it shouldn’t, but it already does it, so I would at least make the default styling reasonable. It is not a blocker for this PR at all

Reply:
I think you may be thinking about [SocialMarkdown](https://near.social/mob.near/widget/WidgetSource?src=near/widget/SocialMarkdown) widget, which is the wrapper for post previews on near.org. Idk who manages that widget, but they can absolutely put these styles in there (or I can make a PR, but I can't find a public repo for these). The VM holds no CSS and does not apply any stylings, [React-markdown](https://github.com/remarkjs/react-markdown) only renders the markdown -- which brings up a good point -- in documentation, we should ensure a wrapper is utilized with the Markdown primitive.

DevHub didn't have a wrapper widget before, so it was just getting styles from the gateway -- maybe there could be a PR to make the default styling of the discovery-gateway reasonable... but I also feel weary against gateway-imposed stylings...